core/core-configuration-layer.el

```diff
@@ -2089,7 +2089,7 @@ LAYER must not be the owner of PKG."
 
 (defun configuration-layer//configure-package (pkg)
   "Configure PKG object, i.e. call its post-init function."
-  (spacemacs/update-progress-bar)
+  (when dotspacemacs-loading-progress-bar (spacemacs/update-progress-bar))
```

Comment:
this is unnecessary 

https://github.com/syl20bnr/spacemacs/blob/0bbb41ec97198b19e1e44ee604caf56596506332/core/core-progress-bar.el#L53